modules/districtmDMXBidAdapter.js

```diff
@@ -122,14 +147,34 @@ export const spec = {
       obj.id = dmx.bidId;
       obj.tagid = String(dmx.params.dmxid);
       obj.secure = 1;
-      obj.banner = {
-        topframe: 1,
-        w: cleanSizes(dmx.sizes, 'w'),
-        h: cleanSizes(dmx.sizes, 'h'),
-        format: cleanSizes(dmx.sizes).map(s => {
-          return {w: s[0], h: s[1]};
-        }).filter(obj => typeof obj.w === 'number' && typeof obj.h === 'number')
-      };
+      obj.bidfloor = dmx.params.floor || 0;
```

Comment:
Please support the floors module as well as your own bidder params. See https://github.com/prebid/Prebid.js/pull/5329/files for an example

Reply:
support for floor module was added